julia/sync.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+mkdir -p /data/tmp
```

Comment:
Using `$TO` instead fixed path `/data` is better. Refer to [README.md](https://github.com/ustclug/ustcmirror-images#volumes)
```suggestion
mkdir -p $TO/tmp
```

Reply:
Ah, I didn't understand `README.md`.  Thanks for the tip.